docs/en/index.rst

```diff
@@ -266,6 +276,53 @@ This shell can take a few different options:
   - Max Runtime
   - Runtime: Time since the worker started, the worker will finish when Runtime is over Max Runtime value
 
+Failed Jobs
```

Comment:
I wonder if that will get confusing because some queue backends implement their own dead letter queues (for example, [sqs](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html), [rabbitt mq](https://www.rabbitmq.com/dlx.html)). And it seems in those examples a dead letter queue is defined more broadly where here only jobs that are requeued too many times will be failed/dead lettered. Also, when a job is failed, it's not going to be queued on the original queue infrastructure, but a database table instead. I think this does fall within the concept of a dead letter queue though. Let me know what you think.

Reply:
I don't disagree that the terminology would overlap with native message brokers. Using 'failed jobs' is a good term for the abstraction we're trying to provide. I think we should mention the similarities with dead letter queues when we explain the behavior.

Reply:
I kept it as failed jobs naming and added some clarification about the similarities/differences in the documentation.